### PR TITLE
Develop to Release

### PR DESCRIPTION
**Pull request by halmasieh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,77 @@
...
...kills in Excel, VBA, Python, R, JavaScript, SQL Databases, Tableau, Big Data, and Machine Learning. I worked with various datasets during the BootCamp program and according to the content of the projects, I used various tools to analyze the data, design the model and visualize the finding through Tableau and dashboards. In addition, I am a former Assistant Professor of University in the area of Applied Mathematics with a brilliant and valuable background in the field of teaching and scientific research. With this extensive background of Mathematics and Data, I started  my training program at Jarvis by developing various projects and this mentorship helped me to gain skills in increasing code efficiency, debugging, and gaining comprehensive knowledge in Linux/Bash, Git, Java, Docker, RDBMS and Springboot. I am a self-initiated and avid learner with high attention to detail, critical thinking and logical problem-solving. My adaptability allows me to work in many different environments, in addition to speaking with people from various backgrounds. I am looking forward to speaking about my academic and professional background for developing positions mainly in the data analytics field.
```
Comment (jarviscanada) [inline README.md:3]: I recently completed Data Analytics BootCamp from the University of Toronto focused on gaining technical programming skills in Excel, VBA, Python, R, JavaScript, SQL Databases, Tableau, Big Data, and Machine Learning. I worked with various datasets during the BootCamp program and according to the content of the projects, I used various tools to analyze the data, design the model and visualize the finding through Tableau and dashboards. In addition, I am a former Assistant Professor of University in the area of Applied Mathematics with a brilliant and valuable background in the field of teaching and scientific research. With this extensive background in Mathematics and Data, I started my training program at Jarvis by developing various projects. This mentorship helped me increase code efficiency, debugging and gain comprehensive knowledge in Linux/Bash, Git, and Java Docker, RDBMS and Springboot. I am a self-initiated and avid learner with great attention to detail, critical thinking and logical problem-solving. My adaptability allows me to work in many different environments and speak with people from various backgrounds.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,77 @@
...
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, Machine Learning and Data Analytics, Statistics, RDBMS/SQL, Tableau, Agile/Scrum, Git
+
+**Competent:** Pandas, Numpy, Scipy, Scikit-Learn, DHTML/CSS
```
Comment (jarviscanada) [inline README.md:9]: Python, Pandas,

Comment (jarviscanada) [reply README.md:9]: remove DHTML/CSS

File: README.md [hunk 3]
```diff
@@ -1,5 +1,77 @@
...
+
+**Familiar:** R, APIs, Tensorflow/Deep learning, Flask, Google Cloud Platform(GCP)
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh/tree/master/linux_sql)]: Implemented a Monitoring Agent on a node/server that collects the hardware specifications and monitors the cluster node resource usage in real-time within the Linux cluster running on CentOS 7. The collected data include the host's CPU number, total memory, memory free, disk I/O, etc which are stored in a RDBMS database. This Monitoring Agent contains a PostgreSQL server installed within a Docker container that collects information every minute using bash scripts and crontab installed on each node.
```
Comment (jarviscanada) [inline README.md:18]: Implemented a Monitoring Agent on a node/server that collects the hardware specifications and monitors the cluster node resource usage in real-time within the Linux cluster running on CentOS 7. The collected data include the host's CPU number, total memory, memory free, disk I/O, etc., which are stored in an RDBMS database. This Monitoring Agent contains a PostgreSQL server installed within a Docker container that collects information every minute using bash scripts and crontab installed on each node.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,77 @@
...
+- Worked on the Gray Predictive Models for using in HVAC residential
+
+**Assistant Professor, Isfahan University (2003-2015)**:
+- Faculty member of Isfahan University for more than 13 years
+- Developed curricula and delivered course materials
+- Researched with over 20 published papers with more than 528 citations
+- Supervised and advised on dissertation in undergraduate and graduate students
+- Specialized in analyzing and synthesizing applied problems and algorithms
+- Journal Reviewer of International Journals in Mathematics, Engineering, Economic and Science
+- Expertised in all areas of Applied Mathematics including but not limited to Numerical Analysis,  ODEs, PDEs, Finance Mathematical Modeling, Stochastic Calculus, Operational Research, Integral Equations, Linear Algebra, Numerical Integration, and Numerical Derivatives
```
Comment (jarviscanada) [inline README.md:63]: https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer

File: README.md [hunk 5]
```diff
@@ -1,5 +1,77 @@
...
+**Azad University (2005-2010)**, Doctor of Philosophy of Applied Mathematics (Integral Equations), Mathematics
+
+**Yazd University (2001-2003)**, Master of Applied Mathematics (Optimal Control), Mathematics
+
+**Isfahan University (1997-2001)**, Bachelor of Applied Mathematics, Mathematics
+
+
+## Miscellaneous
+- Data Analytics- University of Toronto (2021)
+- A list of my publications can be accessed through [this link](https://github.com/halmasieh/Publications)
```
Comment (jarviscanada) [inline README.md:76]: A list of my publications can be accessed through https://github.com/halmasieh/Publications

**Review by jarviscanada — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (halmasieh) [reply README.md:3]: Done!

File: README.md [hunk 6]
```diff
@@ -1,5 +1,77 @@
...
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, Machine Learning and Data Analytics, Statistics, RDBMS/SQL, Tableau, Agile/Scrum, Git
```
[the remark this answers is not in the document]
Comment (halmasieh) [reply README.md:7]: Done!

File: README.md (hunk 2 above)
Comment (halmasieh) [reply README.md:9]: Done!

File: README.md (hunk 4 above)
Comment (halmasieh) [reply README.md:63]: Done!

File: README.md [hunk 7]
```diff
@@ -1,5 +1,77 @@
...
+
+**Yazd University (2001-2003)**, Master of Applied Mathematics (Optimal Control), Mathematics
+
+**Isfahan University (1997-2001)**, Bachelor of Applied Mathematics, Mathematics
+
+
+## Miscellaneous
+- Data Analytics- University of Toronto (2021)
+- A list of my publications can be accessed through [this link](https://github.com/halmasieh/Publications)
+- Strong research scientist with over 20 published articles with more than 528 citations, [click here](https://scholar.google.ca/citations?hl=en&user=D5P5et4AAAAJ)
```
[the remark this answers is not in the document]
Comment (halmasieh) [reply README.md:77]: Done!

**Pull request merged**